packages/node_modules/@ciscospark/react-component-activity-item-base/src/index.js

```diff
@@ -15,12 +15,14 @@ export default function ActivityItemBase(props) {
     children,
     id,
     isAdditional,
+    isError,
```

Comment:
nitpick: `isError` sounds a bit off ... `hasError` sounds more grammatically natural, but is inconsistent.